core-kotlin-modules/core-kotlin-advanced-3/pom.xml

```
@@ -31,6 +31,22 @@
             <artifactId>classgraph</artifactId>
             <version>4.8.112</version>
         </dependency>
+
+        <dependency>
+            <groupId>org.jetbrains.kotlinx</groupId>
+            <artifactId>kotlinx-coroutines-test</artifactId>
+            <version>1.7.3</version>
+            <type>pom</type>
+        </dependency>
+
+        <dependency>
+            <groupId>org.jetbrains.kotlinx</groupId>
+            <artifactId>kotlinx-coroutines-core</artifactId>
+            <version>1.8.1</version>
+            <type>pom</type>
+            <scope>runtime</scope>
+        </dependency>
```

Let's extract these versions as properties.

See [line 53](https://github.com/Baeldung/kotlin-tutorials/pull/1093/files#diff-a48ca0e87d134b3cb9b495658ac00af9be1323dceb5b963e499351b52b694525R53) for an example of that.
```suggestion
        <dependency>
            <groupId>org.jetbrains.kotlinx</groupId>
            <artifactId>kotlinx-coroutines-test</artifactId>
            <version>${kotlinx-coroutines.version}</version>
            <type>pom</type>
        </dependency>

        <dependency>
            <groupId>org.jetbrains.kotlinx</groupId>
            <artifactId>kotlinx-coroutines-core</artifactId>
            <version>${kotlinx-coroutines.version}</version>
            <type>pom</type>
            <scope>runtime</scope>
        </dependency>
```